config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -325,7 +325,13 @@ periodics:
       - bash
       image: gcr.io/k8s-testimages/kubekins-e2e:v20200802-ff98b58-1.16
       name: ""
-      resources: {}
+      resources:
+        limits:
+          cpu: 4
+          memory: "10Gi"
```

Comment:
@tpepper noticed @spiffxp recommended a little more than 10 here. Were you wanting to try this and then bump up?

Reply:
Aaaah I was reading that as start with 10 and see if we need to go up more to 12 based on results.  I'll switch it to 12.

Reply:
amended